src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -250,6 +250,56 @@ private function isFunction(Tokens $tokens, $index)
         return $tokens[$index]->isGivenKind(T_FUNCTION) && !$tokensAnalyzer->isLambda($index);
     }
 
+    /**
+     * @param string $functionName
+     *
+     * @return bool
+     */
+    private function hasTestPrefix($functionName)
+    {
+        $length = strlen($functionName);
+
+        if ($this->startsWith('test_', $functionName) && $length > 5) {
+            return true;
+        }
+
+        if ($this->startsWith('test', $functionName) && $length > 4) {
+            $nextCharacter = substr($functionName, 4, 1);
+
+            return strtoupper($nextCharacter) === $nextCharacter;
+        }
+
+        return false;
+    }
+
+    /**
+     * @param string $functionName
+     *
+     * @return bool
+     */
+    private function canRemoveTestPrefix($functionName)
+    {
+        $length = strlen($functionName);
+
+        if ($this->startsWith('test_', $functionName) && $length > 5) {
```

Comment:
Maybe add a test case to prove a function just being named ```test```or ```test_``` does not get renamed either

Reply:
@BackEndTea 

Good point, will take a look later!